lbz/exceptions.py

```diff
@@ -15,20 +16,23 @@ class LambdaFWException(Exception):
     message = HTTPStatus.INTERNAL_SERVER_ERROR.description
     status_code = HTTPStatus.INTERNAL_SERVER_ERROR.value
 
-    def __init__(self, message: str = "") -> None:
+    def __init__(self, message: str = "", error_code: Optional[str] = None) -> None:
         super().__init__(message)
         if message:
             self.message = message
+        self.error_code = error_code
 
     def __str__(self) -> str:
-        return f"[{self.status_code}] {self.message}"
+        full_msg = f"{self.error_code} - {self.message}" if self.error_code else self.message
```

Comment:
Personally, I would avoid making double f-string, this is small enough to do something like this:
```python
if self.error_code is not None:
    return f"[{self.status_code}] {self.error_code} - {self.message}"
return f"[{self.status_code}] {self.message}"
```